DataManagementSystem/Agent/RequestOperations/ReplicateAndRegister.py

```diff
@@ -101,9 +109,17 @@ def filterReplicas(opFile, logger=None, dataManager=None):
 
   for repSEName in replicas:
     repSEMetadata = StorageElement(repSEName).getFileMetadata(opFile.LFN)
-    error = repSEMetadata.get('Message', repSEMetadata.get('Value', {}).get('Failed', {}).get(opFile.LFN))
```

Comment:
This line is less than 120 characters, why is this modified?
```
>>> len("    error = repSEMetadata.get('Message', repSEMetadata.get('Value', {}).get('Failed', {}).get(opFile.LFN))")
106
```

Reply:
I have no idea ! But it does it consistently